builder-support/specs/dnsdist.spec

```diff
@@ -144,35 +123,27 @@ exit 0
 %if 0%{?suse_version}
 %service_add_post %{name}.service
 %endif
-%if 0%{?rhel} >= 7
```

Comment:
with the suse context above, this looks like this block **might** have only been in place for rhel?